ext/JuMPExt.jl

```diff
@@ -0,0 +1,142 @@
+module JuMPExt
+
+using StaticArrays: StaticArray, SVector
+using JuMP: @variable, @objective, @constraint, print, set_silent,
+            optimize!, is_solved_and_feasible, value, set_string_names_on_creation,
+            set_objective_coefficient, @expression
+using SciMLBase: DiscreteCallback
+using PositiveIntegrators
+
+mutable struct SanduProjection{M} <: PositiveIntegrators.SanduProjection
+    model::M
+    cnt::Int
+end
+
+"""
+    SanduProjection(model, AT, b, eps = nothing; [save = true, verbose = false])
+
+A projection method which ensures conservation of prescribed linear invariants and positivity.
+If the current approximation ``\\mathbf{u}`` has negative components then a projection ``\\mathbf{z}`` is computed such that
+```math
+\\min \\lVert \\mathbf{z} - \\mathbf{u} \\rVert_G,\\quad \\mathbf{A}^T\\mathbf{z}=\\mathbf{b},\\quad \\mathbf{z}≥ \\mathbf{0},
```

Comment:
```suggestion
\\min \\lVert \\mathbf{z} - \\mathbf{u} \\rVert_{\\mathbf{G}},\\quad \\mathbf{A}^T\\mathbf{z}=\\mathbf{b},\\quad \\mathbf{z}≥ \\mathbf{0},
```
For consistency with how it's written below.